File: Hospital Management System All/Hospital Management System GUI/Hospital Management System/app/model/Treatment.js
Ext.define('HMS.model.Treatment', {
    extend: 'HMS.model.Base',

    fields: [
        'treatment_id','patient_serial_no','tx_patient_name','patient_age', 'tx_doctor_name', 
        'tx_medicine_names', 'tx_test_names', 'treatment_fee'
    ]
});

File: Hospital Management System All/Hospital Management Api/Hospital Management/src/main/java/com/hospital_management/model/Treatment.java
package com.hospital_management.model;

import jakarta.persistence.*;
import lombok.Data;

@Entity
@Data
@Table(name ="T_H_TREATMENT")
public class Treatment {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int treatment_id;

    private int doctor_id ;
    private int patient_serial_no;
    private String tx_medicine_names;
    private String tx_test_names;
    private int treatment_fee;

}
